### PR TITLE
[RFC] add sitevision-scripts

### PR DESCRIPTION
**Pull request by alexandernanberg**
Refactor to use https://github.com/strt/sitevision/tree/master/packages/sitevision-scripts instead of SiteVisions utils.

This is still very WIP but I wanted to get some opinions on this before I continue.

Main changes
- `sitevision-scripts` will provide all the utils/scripts we need to handle addons. 
- ~`manifest.json` is generated from the `package.json` file (a lot of properties were already duplicated, only hade to add `type` + `displayName`).~
- `.dev-properties.json` is removed in favor of a `.env` file.

Also have some ideas of a `dev` command that will watch the `src` files for changes and build + deploy them directly.

Let me know what you think 😄

### CODE REVIEW

**Review by jesperjohansson — CHANGES_REQUESTED** [review]
Provide information in the readme.

**Comment by alexandernanberg** [comment]
@jesperjohansson I think if this get accepted that we should move away from this fork and create new repo. A readme would be created/updated then instead

File: package.json [hunk 1]
```diff
@@ -1,54 +1,42 @@
...
     "prettier-eslint-cli": "^4.7.1",
-    "request": "^2.87.0",
-    "zip-dir": "^1.0.2"
-  },
-  "lint-staged": {
-    "src/**/*.js": [
-      "eslint --config '.eslintrc' --fix",
-      "git add"
-    ]
+    "sitevision-scripts": "^0.1.0"
```
Comment (alexpkarlsson) [inline package.json]:
```suggestion
    "sitevision-scripts": "^0.3.0"
```

Bump to most recent version.

File: package.json [hunk 2]
```diff
@@ -1,54 +1,42 @@
 {
+  "type": "RESTApp",
   "name": "restapp-boilerplate",
```
Comment (alexpkarlsson) [inline package.json]: Add `displayName` as well to signal that it should exist.

File: package.json [hunk 3]
```diff
@@ -1,54 +1,42 @@
...
   ],
-  "bugs": {
-    "url": "https://github.com/strt/restapp-boilerplate/issues"
+  "scripts": {
+    "build": "sitevision-scripts build",
+    "setup": "sitevision-scripts setup",
+    "deploy": "sitevision-scripts deploy",
+    "create-addon": "sitevision-scripts create-addon",
+    "sign": "sitevision-scripts sign",
+    "prod-deploy": "echo 'todo'"
```
Comment (alexpkarlsson) [inline package.json]:
```suggestion
    "prod-deploy": "sitevision-scripts deploy --production"
```

Implement new `--production` flag added in `0.3.0`.

**Review by alexpkarlsson — CHANGES_REQUESTED** [review]
Some minor changes. Also update `yarn.lock` to new `sitevision-scripts` version.

File: package.json (hunk 3 above)
Comment (alexpkarlsson) [reply package.json]: Actually, might be better to just remove this and force the use to simply run `deploy --production`?

**Review by alexpkarlsson — APPROVED** [review]

**Review by jesperjohansson — APPROVED** [review]

**Pull request merged**